setup.py

```diff
@@ -193,7 +190,7 @@ def git_version(version_: str) -> str:
         if repo.is_dirty():
             return f".dev0+{sha}.dirty"
         # commit is clean
-        return f".release:{version_}+{sha}"
+        return f".release:{sha}"
```

Comment:
This changes format of the git tag, but given this is shown directly beneath the actual version in the UI it seemed easier to change it this way, rather than have to have this do the ast eval to get version too.

If anyone thinks otherwise I can change the style back